README.md

```diff
@@ -1,23 +1,44 @@
-# purescript-arraybuffer-types
+# ArrayBuffer Types
 
-[![Latest release](http://img.shields.io/github/release/purescript-contrib/purescript-arraybuffer-types.svg)](https://github.com/purescript-contrib/purescript-arraybuffer-types/releases)
-[![Build status](https://travis-ci.org/purescript-contrib/purescript-arraybuffer-types.svg?branch=master)](https://travis-ci.org/purescript-contrib/purescript-arraybuffer-types)
-[![Pursuit](http://pursuit.purescript.org/packages/purescript-arraybuffer-types/badge)](http://pursuit.purescript.org/packages/purescript-arraybuffer-types/)
-[![Maintainer: garyb](https://img.shields.io/badge/maintainer-garyb-lightgrey.svg)](http://github.com/garyb)
-[![Maintainer: thomashoneyman](https://img.shields.io/badge/maintainer-thomashoneyman-lightgrey.svg)](http://github.com/thomashoneyman)
+[![CI](https://github.com/purescript-contrib/purescript-arraybuffer-types/workflows/CI/badge.svg?branch=main)](https://github.com/purescript-contrib/purescript-arraybuffer-types/actions?query=workflow%3ACI+branch%3Amain)
+[![Release](https://img.shields.io/github/release/purescript-contrib/purescript-arraybuffer-types.svg)](https://github.com/purescript-contrib/purescript-arraybuffer-types/releases)
+[![Pursuit](https://pursuit.purescript.org/packages/purescript-arraybuffer-types/badge)](https://pursuit.purescript.org/packages/purescript-arraybuffer-types)
+[![Maintainer: garyb](https://img.shields.io/badge/maintainer-garyb-teal.svg)](https://github.com/garyb)
+[![Maintainer: thomashoneyman](https://img.shields.io/badge/maintainer-thomashoneyman-teal.svg)](https://github.com/thomashoneyman)
 
 Type definitions for JavaScript `ArrayBuffer`s.
 
 ## Installation
 
+Install `arraybuffer-types` with [Spago](https://github.com/purescript/spago):
+
+```sh
+spago install arraybuffer-types
 ```
-bower install purescript-arraybuffer-types
-```
+
+## Quick start
+
+The quick start hasn't been written yet (contributions are welcome!). The quick start covers a common, minimal use case for the library, whereas longer examples and tutorials are kept in the [docs directory](./docs).
 
 ## Documentation
 
-Module documentation is [published on Pursuit](http://pursuit.purescript.org/packages/purescript-arraybuffer-types).
+`arraybuffer-types` documentation is stored in a few places:
+
+1. Module documentation is [published on Pursuit](https://pursuit.purescript.org/packages/purescript-arraybuffer-types).
+2. Written documentation is kept in the [docs directory](./docs).
+3. Usage examples can be found in [the test suite](./test).
```

Comment:
We can remove parts of this section which aren't relevant for a library (for example, this one doesn't have tests and so we should remove that line. It probably doesn't need a `docs` directory, either).

Reply:
Other than that, this looks good!

Reply:
Alright, I removed the `docs` directory and the references to `docs` and `test` in the README.